conf_general.h

```diff
@@ -209,7 +216,7 @@
 
 // CAN-plotter
 //#define APP_CUSTOM_TO_USE			"app_plot_can.c"
-//#define APPCONF_APP_TO_USE			APP_CUSTOM
+#define APPCONF_APP_TO_USE			APP_NONE
```

Comment:
APP_NONE should not be selected by default. In most cases it is selected by the hardware file or default config, where UART is the default most of the time.